syntax = "proto3";
import "protos/base.proto";
import "protos/showtime.proto";

service Booking {
    rpc GetAllBookings(Empty) returns (stream BookingData) {}
    rpc GetBookingByUserId(UserID) returns (BookingData) {}
}

message BookingData {
    string userId = 1;
    repeated Schedule date = 2;
}